apis/Google.Bigquery.V2/Google.Bigquery.V2.Snippets/BigqueryClientSnippets.cs

```diff
@@ -585,6 +585,49 @@ public void ListProjects()
             Assert.Contains(_fixture.ProjectId, projects.Select(p => p.ProjectId));
         }
 
+        [Fact]
+        public void ParameterizedQuery_NamedParameters()
```

Comment:
Done - just as a sample for one, and a snippet+sample for the other.
I've kept the tests, but afterwards.